artifactory/artifacts_test.go

```diff
@@ -101,7 +102,11 @@ func Test_Artifacts(t *testing.T) {
 				g.Assert(actual == nil).IsTrue()
 				g.Assert(resp == nil).IsTrue()
 				g.Assert(err != nil).IsTrue()
-				g.Assert(err.Error()).Equal("open this/is/a/bad/path.txt: no such file or directory")
+				if runtime.GOOS == "windows" {
+					g.Assert(err.Error()).Equal("open this/is/a/bad/path.txt: The system cannot find the path specified.")
```

Comment:
Update code to match the below suggestion with Windows specific error type.

Reply:
Sure thing.